src/shared/PortMessageChannel.ts

```diff
@@ -43,6 +44,12 @@ export class PortMessageChannel {
     });
     this.emitter.on('message', (response) => {
       if (isJsonRpcPayload(response)) {
+        if (isJsonRpcError(response)) {
+          console.group('Port method error'); // eslint-disable-line no-console
```

Comment:
Do we need this in production?

Reply:
Yes, it's similar to what I did here: https://github.com/zeriontech/zerion-wallet-extension/blob/cd1303408519e38fbcc6b60b72e92f21c58752e8/src/background/messaging/mapRPCMessageToController.ts#L60-L62

It's useful both for debugging during dev and to investigate user reports. Otherwise there used to be no way to understand which requests between the UI layer and background caused errors